documentation/modules/cruise-control/con-rebalance-performance.adoc

```diff
@@ -75,7 +75,7 @@ The relevant configurations are summarised below:
 
 | `default.replica.movement.strategies`             .2+| 
   The list of strategies (in priority order) used to determine the order in which partition reassignment commands are executed for generated proposals. Use a comma separated string for server and a YAML array for `KafkaRebalance` resources.
-.2+| `BaseReplicaMovementStrategy`
+.2+|Base Replica Movement Strategy
```

Comment:
No it needs to be `BaseReplicaMovementStrategy` as that is the name of the Java class that will be used.

Reply:
OK, I'll change it back. I attempted to wrap the string using Asciidoctor line breaks, but no luck.